app/src/migration/java/org/mozilla/fenix/MigratingFenixApplication.kt

```diff
@@ -7,6 +7,7 @@ package org.mozilla.fenix
 import android.content.Context
 import kotlinx.coroutines.runBlocking
 import mozilla.components.support.migration.FennecMigrator
+import org.mozilla.fenix.migration.MigrationTelemetryListener
```

Comment:
Why does this file need the telemetry listener?

Reply:
It [already uses it](https://github.com/NotWoods/fenix/blob/use-main-coroutine-rule/app/src/migration/java/org/mozilla/fenix/MigratingFenixApplication.kt#L53), I'm just fixing the package name here